frontend/src/Frontend/UI/Dialogs/AddVanityAccount.hs

```diff
@@ -144,7 +144,8 @@ uiAddVanityAccountSettings ideL onInflightChange mInflightAcc mChainId initialNo
           $ uiSenderDropdown def never
 
       (mGasPayer, signers, capabilities) <- tabPane mempty curSelection DeploymentSettingsView_Keys $
-        uiSenderCapabilities ideL cChainId Nothing mSender $ uiSenderDropdown def never ideL cChainId
+        uiSenderCapabilities ideL cChainId Nothing mSender
+          $ uiSenderDropdown def (updated mSender) ideL cChainId
```

Comment:
If the user goes back a page and changes the sender, wont this trash what the user previously chose? Is that OK?

Reply:
- Yes
- Not really, no